README.md

```diff
@@ -51,13 +51,13 @@ yarn run dev
 </details>
 <p></p>
 
-Your application is running at http://localhost:3000/.
+Your application will be running at http://localhost:3000/.
```

Comment:
At this point it is running already (yarn run dev runs the app)

Reply:
This looks like a future event to me, at least at the moment of someone reading this.
It's better to ask a corrector or a native speaker.